db/schema.rb

```diff
@@ -95,7 +95,7 @@
     t.bigint "user_id", null: false, unsigned: true
     t.string "hat", null: false
     t.string "link", null: false
-    t.text "comment", size: :medium, null: false
+    t.text "comment", null: false
```

Comment:
this kind of chages happen because:
`t.text "comment", size: :medium, null: false` will result in a `mediumtext` column with a [default size](https://mariadb.com/kb/en/mediumtext/) of (2**24 - 1)

![Screenshot 2024-10-22 at 10 16 53 p m](https://github.com/user-attachments/assets/b00dad37-3aaa-44d0-a8c9-e4b828b39dcb)

since the model is validating for (2**16 - 1) we should have defined the column as:
`t.text "comment", null: false` to have a [proper value](https://mariadb.com/kb/en/text/), this is why the migration define the change as:
`change_column :hat_requests, :comment, :text, limit: 65_535`

![Screenshot 2024-10-22 at 10 20 09 p m](https://github.com/user-attachments/assets/7fa57877-30e3-443f-b0cb-c3fad63a9117)